src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -14,6 +14,7 @@ Rectangle {
     property alias chatInfoButton: statusChatInfoButton
     property alias menuButton: menuButton
     property alias notificationButton: notificationButton
+    property alias notificationTooltip: notificationButton.tooltip
```

Comment:
@PascalPrecht  This is something I am not happy with. if any of the other buttons tooltip need modification for future features, then we need to expose each of these individually. if instead all these button came form a common file such as the 
-> "StatusFlatRoundButtonWithTooltip" or 
-> adding the tooltip in the StatusFlatRoundButton and it will only be visible if someone adds text for the tooltip. w

then these could be exposed directly whenw e expose the button from the StatusChatToolBar

Reply:
Okay so I actually thought we should expose the tooltip on the `notificationButton` which is already exposed on the chat toolbar.

That way no new property in chat toolbar is needed. Again, the end result would be:

```
StatusChatToolBar {
  notificationButton.tooltip.[...] = ...
}
```
This can be done for all buttons exposed by the toolbar to be consistent.

As discussed offline, we can explore the idea of adding tooltips to `StatusBaseButton` so all buttons get them for free. But that's a new feature which is not scope of this PR.

Reply:
This property can now be removed :)

Reply:
Done!